### ISSUE TITLE
Partial rendering breaks style tags

### ISSUE THREAD

**Issue by mc680x0** [stolksdorf/naturalcrit#77]
**Browser Type/Version**: [Google Chrome 49.0.2623.87 (Official Build) m (32-bit)]

**Operating System**: [Windows 7 x64 Professional]

**Issue Description**: [Regression in application of CSS to brew, possibly due to Partial Page Rendering]

I believe this is a regression due to the changelog indicating that page rendering was altered with release 1.4, and the issue appearing only after the update to 1.4 

**Markdown code to reproduce**:

source of http://www.naturalcrit.com/homebrew/share/EJ9htm-3l - I can't figure out a good way to isolate to a test case. 

**Steps to reproduce**: 

Scroll until styles appear to no longer be in effect. This can be reproduced on the latest google-chrome-stable both on Windows and on Gentoo Linux. 

**Related Images** :

The imgur album http://imgur.com/a/QQm5K shows two images - one where the "ink friendly" and "A4" styles are in effect and visibly working, and another where they've for some reason ceased to function despite still being present in the brew. (Note: I've disabled the chunk of "ink friendly" that removes images, only the background pattern is removed.)

**Comment by Tsurumah** [comment]
I've noticed if you put the code for, example, the "ink friendly" option every two "pages", it sets it back up.

**Comment by mc680x0** [comment]
That's ... a suboptimal solution, but I suppose it'll let me continue work until this is sorted out properly.

**Comment by Tsurumah** [comment]
Yeah, but at least it works for now.

**Comment by stolksdorf** [comment]
Ah, I understand what's happening. I implemented partial page rendering, and so when the page with the style tags is de-rendered, the styles stop being picked up. 

Huh, interesting problem. I just reverted back to v1.3 until I fix the other problems. I'll add this to my test cases to check against for v1.5

My current idea to fix this is to detect if a page has style tags on it and if so, always render it. This is a quick and dirty solution, but most brews use style tags on one page only, so it shouldn't effect render times.

Thanks!
